app/controllers/claims_controller.rb

```diff
@@ -4,7 +4,7 @@ class ClaimsController < BasePublicController
   skip_before_action :send_unstarted_claimants_to_the_start, only: [:new, :create]
   before_action :initialize_session_slug_history
   before_action :check_page_is_in_sequence, only: [:show, :update]
-  before_action :update_session_with_current_slug, only: [:update]
+  after_action :update_session_with_current_slug, only: [:update]
```

Comment:
i'd recommend moving this a few lines down so the `after_action` is not in a midst of `before_action`s